downstream/assemblies/platform/assembly-ug-controller-notifications.adoc

```diff
@@ -0,0 +1,43 @@
+[id="controller-notifications"]
+
+= Notifications in {ControllerName}
+
+A [ADD XREF]Notification Template is an instance of a [ADD XREF]Notification type which includes Email, Slack, or a Webhook, with a name, description, and a defined configuration.
+
+The following include examples of details needed to add a notification template:
+
+* A username, password, server, and recipients are needed for an Email notification template
+* The token and a list of channels are needed for a Slack notification template
+* The URL and Headers are needed for a Webhook notification template
+
+A notification is a manifestation of the notification template.
+For example, when a job fails, a notification is sent using the configuration that you define in the notification template.
```

Comment:
So this line explains all you need to know.

remove the 'for example,' though.